src/cryptoadvance/specter/api/rest/jwt.py

```diff
@@ -0,0 +1,58 @@
+import jwt
+from flask import current_app as app
+from cryptoadvance.specter.api.rest.base import (
+    BaseResource,
+    rest_resource,
+    AdminResource,
+)
+import uuid
+import datetime
+import logging
+from ...user import *
+from .base import *
+
+from .. import auth
+
+logger = logging.getLogger(__name__)
+
+
+def generate_jwt(user):
+    payload = {
+        "user": user.username,
+        "exp": datetime.datetime.utcnow() + datetime.timedelta(days=1),
+    }
+    return jwt.encode(payload, app.config["SECRET_KEY"], algorithm="HS256")
+
+
+@rest_resource
+class TokenResource(AdminResource):
+    endpoints = ["/v1alpha/token/"]
+
+    def get(self):
```

Comment:
The REST standard defines that a get-request does not change the state of the resource but in that case, you're creating a token. So please make change it to a post-request. As the token should only be handed out once. The corresponding get-reuqest should probably return a 404 if the token does not exist and an empty response if it does.

Reply:
> The REST standard defines that a get-request does not change the state of the resource but in that case, you're creating a token. So please make change it to a post-request. As the token should only be handed out once. The corresponding get-reuqest should probably return a 404 if the token does not exist and an empty response if it does.
> 
> I think it makes sense to store more than one token, just right from the beginning. So maybe you design the REST endpoint for that. On the user-side, maybe store them in a hash so that each token gets an ID which needs to be specified by the user with the post-reuqest.

@k9ert I've resolved the first part of the review and have added the second part in the recent push, kindly check ;)